schema/table.go

```diff
@@ -504,9 +504,9 @@ func (t *Table) ValidateColumnNames() error {
 }
 
 func (t *Table) Column(name string) *Column {
-	for _, c := range t.Columns {
-		if c.Name == name {
-			return &c
+	for i := range t.Columns {
```

Comment:
Can this cause unwanted mutations to the column? If the caller assumes it's a copy and mutates the column this might has unexpected results.
What's the reason this is needed?

Reply:
We need it to add `pk` setting to the `id` column in the extensive delete-stale test.

IMO, if we have a pointer returned from the func (not the value) we intend the value to be mutable.

Reply:
However, I see your point, I'll revert the change & set the option in a different manner.
I think in the next SDK version we should make it `func (t *Table) Column(string) (Column, bool)` to distinguish the missing value separately, as the pointer seemed as an intention to  allow mutating the value.

Reply:
done in aa6662099adfdb5f255b5d0c5712b598d3f2b719

Reply:
Yeah, mostly trying to be careful with the impact that is hard to predict even if the intention was to return a mutable column